butterfree/pipelines/feature_set_pipeline.py

```diff
@@ -137,11 +139,13 @@ def __init__(
         feature_set: FeatureSet,
         sink: Sink,
         spark_client: Optional[SparkClient] = None,
+        delta_config: Optional[DeltaConfig] = None,
```

Comment:
This seems like a weird place for this config 🤔 Since this is directly related to the writing part of the ETL, it should be better placed close to the Sink (probably as a custom `HistoricalFeatureStoreWriter`?)